docs/enhancements/backup-restore-support.md

```diff
@@ -0,0 +1,151 @@
+
+#### Story 2
+
+As an ACM/OADP user, I want to backup an installed spoke cluster from an active hub.
+Then, I want to safely restore the cluster on a passive hub, without including the BMHs, and enable a GitOps tool (e.g ArgoCD) on the passive hub. If the GitOps tool is configured to apply the BMHs CRs, it should not affect the spoke cluster.
+
+#### Stroy 3
+
+As a ZTP user, I want to safely move installed spoke clusters between hubs. Either manually (by applying CRs), or using a GitOps tool (e.g. ArgoCD). The spoke cluster on the target hub should behave the same and keep the status of BMHs.
+
+#### Story 4
+
+As an ACM/OADP or ZTP user, I want to restore/move a spoke cluster in non-installed status (draft/in-progress/etc). The cluster on target hub should, ideally, keep the same status. Or, at least, fail gracefully.
+
+### Implementation Details/Notes/Constraints
+
+#### Solution for BMHs reconcile (due to a missing status)
+
+According to BMO documentation, the suggested approach for moving BareMetalHost from one cluster to another is using the `baremetalhost.metal3.io/status` [annotation](https://github.com/metal3-io/metal3-docs/blob/main/docs/user-guide/src/bmo/status_annotation.md).
+I.e. Extract the current status of the BMH, store it in an annotation, create the BMH on the new hub:
+```
+# Save the status in json format to a file
+kubectl get bmh <name-of-bmh> -o jsonpath="{.status}" > status.json
+# Save the BMH and apply the status annotation to the saved BMH.
+kubectl -n metal3 annotate bmh <name-of-bmh> \
+  baremetalhost.metal3.io/status="$(cat status.json)" \
+  --dry-run=client -o yaml > bmh.yaml
+```
+This flow ensures that the BMH on the new hub has status (BMO applies the status using the annotation). Hence, as the BMH should be in state 'Provisioned', the BMH reconcile is avoided.
+I.e. prevent destruction of restored spoke clusters.
+
+##### Suggested solution for automating the flow (make it transparent to the user):
+
+For BMHs in the active hub: (In assisted-service -> `bmh_agent_controller`)
+* Once the BMH is in 'Provisioned' state:
+  * Extract BMH's status and add `baremetalhost.metal3.io/status` to the object metadata.
+  * Add also, add the `baremetalhost.metal3.io/paused` annotation:
+    * This is required since once the annotation is added, BMO updates the BMH's status and delete the annotation.
+    * The [paused](https://github.com/metal3-io/baremetal-operator/blob/main/docs/api.md#pausing-reconciliation) annotation, is similar to the `detached` annotation we're already using - but, it's also pausing the reconcile loop. Thus, keeping the status annotation intact.
+
+For BMHs in the passive hub: (In assisted-service -> `bmh_agent_controller`)
```

Comment:
I think I understand this section, but starting it with knowledge about which hub we're on is a bit confusing since a controller would not know this.

It might be better to format this as explaining the logic from the controller point of view. Something like:

If a BMH is finished installing and has status, apply the paused annotation and apply the status in the status annotation.
If a BMH is missing status and has the status and paused annotations, remove the paused annotation to allow the status to apply.

Does that cover all the logic?